From 003e570725270e0ebbe0f70bbcb756e67d1048bd Mon Sep 17 00:00:00 2001
From: Andrew Purtell <apurtell@apache.org>
Date: Thu, 16 Jul 2015 18:27:30 -0700
Subject: [PATCH 261/295] HBASE-14050 NPE in
 org.apache.hadoop.hbase.ipc.RpcServer.readAndProcess

Reason: Bug
Author: Andrew Purtell
Ref: CDH-31924
---
 .../org/apache/hadoop/hbase/ipc/RpcServer.java     |    1 -
 1 file changed, 1 deletion(-)

diff --git a/hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/RpcServer.java b/hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/RpcServer.java
index f62131e..b08c9b9 100644
--- a/hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/RpcServer.java
+++ b/hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/RpcServer.java
@@ -1807,7 +1807,6 @@ public class RpcServer implements RpcServerInterface {
     protected synchronized void close() {
       disposeSasl();
       data = null;
-      this.dataLengthBuffer = null;
       if (!channel.isOpen())
         return;
       try {socket.shutdownOutput();} catch(Exception ignored) {} // FindBugs DE_MIGHT_IGNORE
-- 
1.7.9.5

